### ISSUE TITLE
Throws “RangeError: Invalid string length” with node 4

### ISSUE THREAD

**Issue by gurdiga** [evanshortiss/lintspaces-cli#6]
In Node 4 `process.argv[0]` is `'/usr/local/bin/node'` and not just `'node'` as in 0.12.7, and because of that, the filter function at https://github.com/evanshortiss/lintspaces-cli/blob/master/index.js#L80 returns `true` and now `lintspaces` ends up trying to lint the Node binary, and fails with `RangeError: Invalid string length`. :-)

A quick hack would be to add `.slice(1)`, like this:
```patch
diff --git a/index.js b/index.js
index e10d276..8be7cac 100644
--- a/index.js
+++ b/index.js
@@ -77,7 +77,7 @@ validator = new Validator({
 
 
 // Get files from args to support **/* syntax. Probably not the best way...
-targetFiles = process.argv.filter(function (f) {
+targetFiles = process.argv.slice(1).filter(function (f) {
   if (f === process.argv[1]) {
     return false;
   } else {
```

@evanshortiss What do you think? 8-)

**Comment by evanshortiss** [comment]
I'd be happy with this, if you want to make a Pull Request I'll be happy to merge @gurdiga

**Comment by evanshortiss** [comment]
Patched in 0.1.1 thanks to @gurdiga
